deployment/deployment.json

```diff
@@ -4617,49 +4617,49 @@
           }
         }
       }
-    },
-    "tornado-cash": {
-      "schema": "generic",
-      "base": "tornado-cash",
-      "protocol": "tornado-cash",
-      "deployments": {
-        "tornado-cash-mainnet": {
-          "network": "mainnet",
-          "status": "prod",
-          "versions": {
-            "schema": "1.3.0",
-            "subgraph": "1.0.0",
-            "methodology": "1.0.0"
-          },
-          "deployment-ids": {
-            "hosted-service": "tornado-cash-mainnet"
-          },
-          "files": {
-            "template": "tornado-cash.template.yaml"
-          },
-          "options": {
-            "prepare:yaml": true,
-            "prepare:constants": true
-          }
-        },
-        "tornado-cash-bsc": {
-          "network": "bsc",
-          "status": "prod",
-          "versions": {
-            "schema": "1.3.0",
-            "subgraph": "1.0.0",
-            "methodology": "1.0.0"
-          },
-          "deployment-ids": {
-            "hosted-service": "tornado-cash-bsc"
-          },
-          "files": {
-            "template": "tornado-cash.template.yaml"
-          },
-          "options": {
-            "prepare:yaml": true,
-            "prepare:constants": true
-          }
+    }
+  },
+  "tornado-cash": {
+    "schema": "generic",
+    "base": "tornado-cash",
+    "protocol": "tornado-cash",
+    "deployments": {
+      "tornado-cash-mainnet": {
+        "network": "mainnet",
+        "status": "prod",
+        "versions": {
+          "schema": "1.3.0",
+          "subgraph": "1.0.0",
+          "methodology": "1.0.0"
+        },
+        "deployment-ids": {
+          "hosted-service": "tornado-cash-mainnet"
+        },
+        "files": {
+          "template": "tornado-cash.template.yaml"
+        },
+        "options": {
+          "prepare:yaml": true,
+          "prepare:constants": true
+        }
+      },
+      "tornado-cash-bsc": {
+        "network": "bsc",
+        "status": "prod",
+        "versions": {
+          "schema": "1.3.0",
+          "subgraph": "1.0.0",
+          "methodology": "1.0.0"
+        },
+        "deployment-ids": {
+          "hosted-service": "tornado-cash-bsc"
```

Comment:
Since you are at it 🙈 
We also had the old `deployment-ids` instead of `services` for TC

Reply:
Good catch - updated!